lib/kvdb/ikvdb.c

```diff
@@ -1315,6 +1384,12 @@ ikvdb_open(
 
     ikvdb_wal_replay_info_init(self, seqno, gen, txhorizon, &rinfo);
 
+    if (self->ikdb_pmem_only) {
```

Comment:
I think I would rather see an enum to describe pmem-only.

Reply:
Not sure I follow why we need an enum? What is the issue with having a bool to indicate this?